Packages/com.unity.inputsystem/CHANGELOG.md

```diff
@@ -9,7 +9,10 @@ Due to package verification, the latest version below is the unpublished version
 however, it has to be formatted properly to pass verification tests.
 
 ## [Unreleased] - yyyy-mm-dd
-- Fixed Composite binding isn't triggered after ResetDevice() called during Action handler [ISXB-746](https://issuetracker.unity3d.com/product/unity/issues/guid/ISXB-746)
+
+### Fixed
+- Avoid potential crashes from NullReferenceException in FireStateChangeNotifications.
+- Fixed Composite binding isn't triggered after ResetDevice() called during Action handler [ISXB-746](https://issuetracker.unity3d.com/product/unity/issues/guid/ISXB-746).
```

Comment:
I would suggest rewriting this to explain the actual problem in a more readable sentence than copying the subject of the ticket, e.g. 
"Fixed an issue where a composite binding would not be consecutively triggered after `ResetDevice()` has been called from the associated action handler." + link as currently.

Reply:
Maybe not part of your PR but would be good to address. Also noticed fixe lines below lack full stop at the end.